test/functional/apps/management/index.ts

```diff
@@ -31,10 +34,13 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       loadTestFile(require.resolve('./_index_pattern_results_sort'));
       loadTestFile(require.resolve('./_index_pattern_popularity'));
       loadTestFile(require.resolve('./_kibana_settings'));
-      loadTestFile(require.resolve('./_scripted_fields'));
       loadTestFile(require.resolve('./_scripted_fields_preview'));
       loadTestFile(require.resolve('./_mgmt_import_saved_objects'));
       loadTestFile(require.resolve('./_index_patterns_empty'));
+      // Remove this flag when ci doesn't run on OSS
+      if (!isOss) {
+        loadTestFile(require.resolve('./_scripted_fields'));
```

Comment:
qq: is it safe to skip the suite due to only test for lens?

Reply:
I share your concerns my dear @sulemanof . I moved the flag inside, to only skip the tests that have to do with the visualize feature.